tests/src/test/java/alluxio/server/ft/journal/MultiMasterJournalTest.java

```diff
@@ -48,17 +41,7 @@ public void testCheckpointReplay() throws Exception {
     for (int i = 0; i < 10; i++) {
       mCluster.getClient().createFile(new AlluxioURI("/" + i)).close();
     }
-    UfsJournal journal = new UfsJournal(URIUtils.appendPathOrDie(JournalUtils.getJournalLocation(),
-        Constants.FILE_SYSTEM_MASTER_NAME), new NoopMaster(""), 0);
-    CommonUtils.waitFor("checkpoint to be written", () -> {
-      UfsJournalSnapshot snapshot;
-      try {
-        snapshot = UfsJournalSnapshot.getSnapshot(journal);
-      } catch (IOException e) {
-        throw new RuntimeException(e);
-      }
-      return !snapshot.getCheckpoints().isEmpty();
-    });
+    IntegrationTestUtils.waitForCheckpoint(Constants.FILE_SYSTEM_MASTER_NAME);
```

Comment:
How come this didn't fail before?

Reply:
this is a multi-master test, so there isn't any secondary master involved. Backup masters work corectly